android/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
   package="app.lunes">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.VIBRATE"/>
```

Comment:
Don't we need the camera permission as well? https://react-native-camera.github.io/react-native-camera/docs/installation.html#android---other-required-steps

Reply:
weird, why is this working then?

Reply:
No idea. I think we don't need the vibrate permission though, do we?

Reply:
No, i removed it.

Reply:
Maybe because it's a debug build but not sure.